build/templates/attributes.rst.mako

```diff
@@ -28,6 +28,21 @@ table_contents = [
          ]
 table = helper.as_rest_table(table_contents)
 
+rep_cap_attr_desc = '''
+This attribute requires repeated capabilities (usually channels). If set or get directly on the 
```

Comment:
Use "property" since this refers to the Python API property session.foo. Also should not say "requires".

Reply:
Updated